docs/source/getting_started.rst

```diff
@@ -0,0 +1,185 @@
+Getting Started
+============
+
+
+Glossary
+--------
+
+- A ``DataSource`` represents a way to retrieve data from database. Typically, this corresponds to a table in the database. Yet, it could also be a more elaborate object. See the section on 'Alternative ``DataSource`` s' for more detail.
+
+- A ``Constraint`` captures a concrete expectation between either two ``DataSource`` s or a single ``DataSource`` and a reference value.
+
+- A ``Requirement`` captures all ``Constraint`` s between two given ``DataSource`` s or all ``Constraint`` s within a single ``DataSource``. If a ``Requirement`` refers links to two ``DataSource`` s, it is a ``BetweenRequirement``. If a ``Requirement`` merely refers to a single ``DataSource``, it is a ``WithinRequirement``.
+
+- Conceptually, a 'specification' captures all ``Requirement`` s against a database. In practice that means it is usually a separate python file which:
+  - gathers all relevant ``Requirement`` s
+  - turns these ``Requirement`` s' ``Constraint`` s into individual tests
+  - can be used to be called with pytest
```

Comment:
I feel like this sentence has grammar issues.

Reply:
So do I! :)